test/functional/test_site/expected/_markbind/plugins/testMarkbindPlugin.js

```diff
@@ -8,4 +8,6 @@ module.exports = {
     $('#test-markbind-plugin').append(`${pluginContext.post}`);
     return $.html();
   },
+  getLinks: (content, pluginContext, frontMatter, utils) => [utils.buildStylesheet('STYLESHEET_LINK')],
+  getScripts: (content, pluginContext, frontMatter, utils) => [utils.buildScript('SCRIPT_LINK')],
```

Comment:
shall we write a test case for inline scripts?

Reply:
I've added the additional test cases :)